SafeguardSessions/SafeguardSessions.proj

```diff
@@ -1,43 +1,71 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="BuildMez">
+  <!-- SafeguardCommon properties  -->
+  <PropertyGroup> 
+    <RepositoryPath>$([System.IO.Path]::GetDirectoryName($(MSBuildProjectDirectory)))\</RepositoryPath>
+    <SafeguardCommonPath>$(RepositoryPath)\SafeguardCommon\</SafeguardCommonPath>
+    <SafeguardCommonSourcePath>$(SafeguardCommonPath)\Src\</SafeguardCommonSourcePath>
+    <CommonModulesPath>$(SafeguardCommonSourcePath)Modules\</CommonModulesPath>
+    <CommonErrorPath>$(CommonModulesPath)Error\</CommonErrorPath>
+    <CommonRequestPath>$(CommonModulesPath)Request\</CommonRequestPath>
+  </PropertyGroup>
+  <!-- SafeguardSessions properties  -->
   <PropertyGroup>
-    <OutputPath Condition="'$(OutputPath)' == ''">$(MSBuildProjectDirectory)\bin\</OutputPath>
-    <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(MSBuildProjectDirectory)\obj\</IntermediateOutputPath>
+    <SrcPath>$(MSBuildProjectDirectory)\Src\</SrcPath>
+    <ModulesPath>$(SrcPath)Modules\</ModulesPath>
+    <ErrorPath>$(ModulesPath)Error\</ErrorPath>
+    <RequestPath>$(ModulesPath)Request\</RequestPath>
+    <SchemaPath>$(ModulesPath)Schema\</SchemaPath>
+    <DefinitionsPath>$(Schemapath)Definitions\</DefinitionsPath>
+    <VersionPath>$(ModulesPath)Version\</VersionPath>
+    <ResourcePath>$(SrcPath)\Res\</ResourcePath>
+    <AssetsPath>$(ResourcePath)Assets\</AssetsPath>
+    <IconsPath>$(AssetsPath)Icons\</IconsPath>
+  </PropertyGroup>
+  <!-- Deploy properties  -->
+  <PropertyGroup> 
+    <DeployPath>$(MSBuildProjectDirectory)\Deploy\</DeployPath>
+    <DeployedConnectorPath>$(DeployPath)OneIdentitySafeguard\</DeployedConnectorPath>
+    <OutputPath Condition="'$(OutputPath)' == ''">$(DeployedConnectorPath)Bin\</OutputPath>
+    <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(DeployedConnectorPath)Obj\</IntermediateOutputPath>
     <MezIntermediatePath>$(IntermediateOutputPath)MEZ\</MezIntermediatePath>
     <MezOutputPath>$(OutputPath)OneIdentitySafeguard.mez</MezOutputPath>
-    <AssetsPath>$(MSBuildProjectDirectory)\assets\</AssetsPath>
-    <IconsPath>$(AssetsPath)\icons\</IconsPath>
-    <ModulesPath>$(MSBuildProjectDirectory)\modules\</ModulesPath>
-    <RequestPath>$(ModulesPath)\request\</RequestPath>
-    <ErrorPath>$(ModulesPath)\error\</ErrorPath>
-    <SchemaPath>$(ModulesPath)\schema\</SchemaPath>
-    <VersionPath>$(ModulesPath)\version\</VersionPath>
-    <DefinitionsPath>$(Schemapath)\definitions\</DefinitionsPath>
   </PropertyGroup>
-  <ItemGroup>
-    <MezContent Include="SafeguardSessions.pq" />
+  <ItemGroup Label="SafeguardCommon">
```

Comment:
Common code will be introduced in the the next commit. It should be there.

Reply:
ACK.